bliss/encoder/encoder.py

```diff
@@ -271,51 +272,46 @@ def update_metrics(self, batch):
         matching = self.matcher.match_catalogs(target_cat, sample_cat)
         self.sample_metrics.update(target_cat, sample_cat, matching)
 
-    def plot_sample_images(self, batch, logging_name):
-        """Log a grid of figures to the tensorboard."""
-        target_cat = TileCatalog(self.tile_slen, batch["tile_catalog"])
-        target_cat = target_cat.filter_tile_catalog_by_flux(min_flux=self.min_flux_threshold)
-        target_cat_cropped = target_cat.symmetric_crop(self.tiles_to_crop)
-        est_cat = self.sample(batch, use_mode=True)
-        mp = self.tiles_to_crop * self.tile_slen
-        fig = plot_detections(batch["images"], target_cat_cropped, est_cat, margin_px=mp)
-        title = f"Epoch:{self.current_epoch}/{logging_name} images"
-        if self.logger:
-            self.logger.experiment.add_figure(title, fig)
-        plt.close(fig)
+        self.plots.update(
+            self.tile_slen,
+            batch,
+            self.min_flux_threshold,
+            self.tiles_to_crop,
+            self.sample,
```

Comment:
We already have drawn a `sample_cat` in this function. Can we pass that instead of `self.sample`?

Reply:
addressed in 6e25cbbaabeb67023d89586940d5abebb6d124fc